business_rules/actions.py

```diff
@@ -58,10 +64,11 @@ def wrapper(func):
         if isinstance(params, dict):
             params_ = [
                 dict(
-                    label=fn_name_to_pretty_label(name),
-                    name=name,
-                    fieldType=field_type,
-                ) for name, field_type in params.items()
+                    label=fn_name_to_pretty_label(key),
+                    name=key,
+                    fieldType=getattr(value, "field_type", value),
+                    defaultValue=getattr(value, "default_value", None)
+                ) for key, value in params.items()
```

Comment:
Will this require to change all actions definitions? 🤔

Reply:
No, for `fieldType` it takes value of the param dict as before or if there's `ActionParam` used  as value instead of type, it will take it's `field_type` attribute. I will add it as a comment